src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocmethod/InputMissingJavadocMethodPublicOnly2.java

```diff
@@ -0,0 +1,120 @@
+////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: 2021
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
We do not use this header for tests anymore, you can remove.

Reply:
Done.

Reply:
Resolved.